### DISCUSSION TITLE
Use a URL variable to fill a field in a table

### DISCUSSION THREAD

**Discussion by taholmes160** [Budibase/budibase#11726] [Help]
Hi guys

I have a "login" page which sends the user to a search / create page along with the users email address as a URL variable like this  
![image](https://github.com/Budibase/budibase/assets/24684691/16971f2f-3c47-4440-802c-c01aaceb170a)

when I create a row using the default create side panel, it looks like this
![image](https://github.com/Budibase/budibase/assets/24684691/4c52aa0c-98db-469c-afe6-55ffbfad638f)

I need the url variable to get saved with each created row in the fk_FamilyID --- I can SELECT the value from the relationship field fk_FamilyID, but I cannot figure out how to get it to be saved to the fk field in the row

Figure 3 is a shot of the relationship 

![image](https://github.com/Budibase/budibase/assets/24684691/43934b54-f17b-4370-8a0b-3294c57c0925)

Can someone please point me in the right direction?

**Comment by ConorWebb96** [comment]
Hey @taholmes160,

So you wish for the variable in the url to be saved as part of each newly created record? Is that correct? I'm not seeing a field were it would make sense for the email within the url to be included.

Is this perhaps meant to be used somehow with the relationship field which you've described?

You could add an `update field value` action and supply the relationship field with the correct id in order to set it after the button has been clicked and the side panel + form has been loaded. If you are using a table block, you'd have to eject this block in order to customise this further.

I hope this makes sense.
